### PR TITLE
CORE-18929 make objects passed into external event executor @CordaSerializable

### PR DESCRIPTION
**Pull request by LWogan**
see https://github.com/corda/corda-runtime-os/pull/5562

All objects passed into the external event request executor must now be @CordaSerializable

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -5,7 +5,7 @@ cordaProductVersion = 5.2.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 34
+cordaApiRevision = 999
```
Comment (mbrkic-r3) [inline gradle.properties]: Is this mistake?

Comment (LWogan) [reply gradle.properties]: its easier when merging down from release when in dev

**Review by mbrkic-r3 — DISMISSED** [review]
LGTM

**Review by mbrkic-r3 — DISMISSED** [review]
re-approving

**Review by thiagoviana — APPROVED** [review]

**Pull request merged**